src/forms/NewJobForm/index.jsx

```diff
@@ -41,11 +45,23 @@ export const NewJobForm = ({ token, coinSymbol }) => {
   useEffect(() => {
     setFormFilled(
       !isEmptyString(fields.title) &&
-        !isEmptyString(fields.description) &&
-        !isEmptyString(fields.budget)
+      !isEmptyString(fields.description) &&
```

Comment:
And these two also :-)

Reply:
Sorry, it's because my prettifier settings. Fixed.